### PR TITLE
Mobile views added

### PR DESCRIPTION
**Pull request by malinasdfghjkl**
Mobile views have been added to
- student dashboard
- all organisations view
- student tutorial view

### CODE REVIEW

**Comment by yashdhanore** [comment]
This looks really good almost a finished product! But still a few finishing touches left I guess : P

I'm only going through this in the mobile view 

- In tutorial page, the text overflows the white container. 


- In sidebar.tsx för logout svg use fillRule and clipRule instead of fill-rule and clip-rule. If you check the console it will show you this error.

- the icons bottom bar in mobile view (sidebar) are not evenly spaced and the armada logo is towards the very left. Maybe in mobile view change them to evenly spaced? 

- The cards in the mobile view look very congested? And some of the text inside them overflows. Maybe instead of having 3 cards in one row we can have 2 cards? 

- also the arrow logos in bottom of dashboard for swiping through cards can be bigger since we don't allow the swipe functionality it should be clearly visible that you have to click those buttons? 
Or how about we implement swiping functionality 👀 (once other things are done obviously)

- one last thing, if you check the organizations view and swipe all the way down, you can only see the top of last row. The information is hidden behind the bottom bar

**Comment by malinasdfghjkl** [comment]
Hi! 
Thank you for your suggestions!
I think I've solved all the things you brought up except the card row suggestion. 
I made a new component with only two cards in one row, but I haven't managed to implement it to only appear in mobile view.

File: src/components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -71,15 +71,79 @@
...
+    width: 100vw;
+    background-color: var(--white);
+    height: 9vh;
+    position: fixed;
+    bottom: 0px;
+    top:auto;
+    border-color: var(--grey-primary);
+    border-width: 1px;
+    border-style: solid;
+    z-index: 3;
```
Comment (yashdhanore) [inline src/components/Sidebar/Sidebar.css:96]: add padding-bottom: 0.5rem;

File: src/components/Sidebar/Sidebar.css [hunk 2]
```diff
@@ -71,15 +71,79 @@
...
+.armada-logo-img{
+  width: 9vw;
+  height: 9vw;
+  margin-left: 15%;
+}
 
+.armada-logo 
+{
+  width: unset;
+  height: unset;
```
Comment (yashdhanore) [inline src/components/Sidebar/Sidebar.css:142]: add margin-left: 1.5rem;

**Review by yashdhanore — COMMENTED** [review]
Looks good! just add the changes i made so the logo doesnt stick to the left in mobile view.

File: src/components/Sidebar/Sidebar.css (hunk 1 above)
Comment (yashdhanore) [reply src/components/Sidebar/Sidebar.css:96]: I'll do this

**Review by yashdhanore — APPROVED** [review]

**Pull request merged**